Add a section on how to re-setup MFA and use recovery codes (#259)
rubygems/guides: setting-up-multifactor-authentication.md

in the _multifactor authentication_ section, you will see a dropdown menu.
    ![Multifactor section at profile edit page](/images/changing_mfa_step1.png){:class="t-img"}
2. Select your intended option, enter OTP access code from your authenticator app and click **Update**

> 

Reviewer: Will it be better to add something like ‘You can refer to section “using recovery codes and re-setup a previously enabled MFA”’ at the end of line 42?

Author: Yes, great idea. I added the referring link.

## Using recovery codes and re-setup a previously enabled MFA

You might be in a situation where you no longer have access to your rubygems account in an

> authenticator app. This can happen if you loose or upgrade your phone or accidentally deleted

Reviewer 2: Typo fix (loose -> lose) + change the first `or` to a `/` so it's a bit clearer how the grouping is intended.

```suggestion
authenticator app. This can happen if you lose/upgrade your phone or accidentally deleted
```

the rubygems account in the authenticator app.

In this situation, you'll need your recovery codes to gain access to your Rubygems account.
Each recovery code can *only be used once* and you'll need *2 recovery codes* to re-setup
a previously enabled MFA rubygems account on a new device or add it to an authenticator app.

1. To login into your account, enter an unused recovery code as the OTP code when prompted.

2. To setup the new device for multi-factor authentication on a previously MFA enabled account,
you'll need to disable MFA and use a recovery code as the OTP code. Then follow the steps in
the enable multi-factor authentication section above to enable MFA using your new device or
add rubygems back to the authenticator app.

Note: The Google Authenticator app only allows an MFA account to be installed on one device

> and there is no backup or cloud sync of the data. So if you loose or upgrade your phone, you'll

Reviewer 2: ```suggestion
and there is no backup or cloud sync of the data. So if you lose or upgrade your phone, you'll
```

> have to re-setup MFA on the new phone. On the other hand, Authy app allows you to use multiple
> devices and backs up/syncs the data in the cloud across devices.

Reviewer 2: A few suggestions in one spot:
1. Re-word the things about backups/sync a bit.
2. Mention Authenticator Plus, since it's another rather popular option that offers multi-device support.

```suggestion
have to set up MFA again on the new phone. On the other hand, the Authy and Authenticator Plus
apps allow you to use multiple devices by providing cloud backups and cross-device sync
capabilities.
```

(If there's some reason to suggest Authy specifically, not both, lmk and I'll change my suggestion to only be about re-wording things.)

Author: Looks good to me! Made the changes
